.gitignore

```diff
@@ -10,6 +10,10 @@
 
 .idea
 .swp
+.classpath
+.project
+.scannerwork
```

Comment:
Ok to add .project, .settings and .classpath, as they are typically used by IDES. However, what about .scannerwork?

Reply:
Fixed in 4d08affe85545b26afed5461585bc8bd710d5a85